ml_perf/flags/9/train.flags

```diff
@@ -7,8 +7,9 @@
 
 # Device specific hyperparameters re: batch size and LR schedules.
 --train_batch_size=4096
---lr_rates=0.01
---lr_rates=0.002
+--lr_rates=0.16
+--lr_rates=0.016
+--lr_rates=0.0016
```

Comment:
can you post your tensorboard logs to examine?  I'd love to see the effect of these rate cuts on the metrics.